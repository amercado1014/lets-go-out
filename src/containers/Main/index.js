import React, { Component } from 'react';
import './styles.css';

export class Main extends Component {
  constructor(props) {
    super(props);

    this.state = {
      searchValue: ''
    };
  }

  render() {
    return (
      <div>
        <h1>{ "Let's Order Out" }</h1>
        <form>
          <input type="text"/>
          <button type="submit">Search</button>
        </form>
      </div>
    );
  }
}

export default Main;
