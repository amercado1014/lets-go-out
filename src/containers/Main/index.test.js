import React from "react";
import { Main } from "../Main";
import { shallow } from "enzyme";

describe("Main", () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Main />);
  });

  it("should math the snapshot", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
