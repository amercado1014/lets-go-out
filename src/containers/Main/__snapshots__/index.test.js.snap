// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main should math the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Main />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h1>
          Let's Order Out
</h1>,
        <form>
          <input
                    type="text"
          />
          <button
                    type="submit"
          >
                    Search
          </button>
</form>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Let's Order Out",
        },
        "ref": null,
        "rendered": "Let's Order Out",
        "type": "h1",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <input
              type="text"
/>,
            <button
              type="submit"
>
              Search
</button>,
          ],
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "type": "text",
            },
            "ref": null,
            "rendered": null,
            "type": "input",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Search",
              "type": "submit",
            },
            "ref": null,
            "rendered": "Search",
            "type": "button",
          },
        ],
        "type": "form",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h1>
            Let's Order Out
</h1>,
          <form>
            <input
                        type="text"
            />
            <button
                        type="submit"
            >
                        Search
            </button>
</form>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Let's Order Out",
          },
          "ref": null,
          "rendered": "Let's Order Out",
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <input
                type="text"
/>,
              <button
                type="submit"
>
                Search
</button>,
            ],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "type": "text",
              },
              "ref": null,
              "rendered": null,
              "type": "input",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Search",
                "type": "submit",
              },
              "ref": null,
              "rendered": "Search",
              "type": "button",
            },
          ],
          "type": "form",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
